com.unity.netcode.gameobjects/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -740,7 +740,7 @@ internal void DespawnAndDestroyNetworkObjects()
 
                         OnDespawnObject(networkObjects[i], shouldDestroy);
                     }
-                    else
+                    else if (networkObjects[i].IsSceneObject != null && !networkObjects[i].IsSceneObject.Value)
```

Comment:
Worth a comment for clarity of 

"IsSceneObject will be null if I've never connected, or will be true (and stay true) if I've connected and am (and always will be) a scene object."